content/faq/dane.md

```diff
@@ -0,0 +1,19 @@
+---
+linktitle: dane
+title: What about DANE?
+date: 2019-05-22T15:37:08-07:00
+weight: 7
+level: secure
+---
+
+DANE, or [DNS-based Authentication of Named Entities](https://tools.ietf.org/html/rfc7672) is an Internet standard which could singularly do what MTA-STS + the preload list would cover combined. It's a downgrade-resistant way for mailservers to discover and publish each other's TLS information using another technology called DNSSEC. DNSSEC is a protocol for signing and verifying statements by owners of domain names.
```

Comment:
I think maybe this should be a different adverb than "singularly"; how about "on its own"?